src/hidapi/windows/hid.c

```diff
@@ -100,9 +100,6 @@ extern "C" {
 
 #include "../hidapi/hidapi.h"
 
-#undef MIN
```

Comment:
Let's revert this file - it's included from an upstream project so we want to minimize local changes. Otherwise this change looks good, thanks!

Reply:
Reverted changes `src/hidapi`.